packages/runtime/src/helpers/files.ts

```diff
@@ -137,8 +139,8 @@ export const moveStaticPages = async ({
     }
   }
   // Move all static files, except error documents and nft manifests
-  const pages = await globby(['**/*.{html,json}', '!**/(500|404|*.js.nft).{html,json}'], {
-    cwd: root,
+  const pages = await globby(['{app,pages}/**/*.{html,json,rsc}', '!**/(500|404|*.js.nft).{html,json}'], {
```

Comment:
Is `rsc` a new extension for React Server components? I thought the externsion was `*.server.js`?

Reply:
Yeah, it's the new data file